internal/dockerfile2dot/build.go

```diff
@@ -70,6 +70,19 @@ func BuildDotFile(simplifiedDockerfile SimplifiedDockerfile, legend bool) string
 
 		graph.AddNode("G", "\""+stage.ID+"\"", attrs)
 
+		// draw layers per stage
+		if layers {
+			graph.AddSubGraph("G", "cluster_"+stage.ID, map[string]string{"label": "Layers stage "+stage.ID})		
```

Comment:
```suggestion
			graph.AddSubGraph("G", "cluster_"+stage.ID, map[string]string{
				"label": getStageLabel(stage),
				"style": "rounded",
			})
```

Reply:
This uses the stage label and rounds the corners of the cluster to make the stages look the same, no matter if a user chooses to display the layers or not.

Reply:
This is an example of the previous suggestion.
![Dockerfile](https://user-images.githubusercontent.com/35694200/168278822-27df15ec-e020-4930-bcba-edb3455aceba.png)

Reply:
Thanks Daniel 🙂